src/hooks/useStory.ts

```diff
@@ -26,8 +30,41 @@ export const useStory = ({ targetId, targetType }: IUserStory) => {
         }
       }
     );
+    return unsubscribe();
+  }, []);
 
-    return () => unsubscribe();
-  }, [targetId, targetType]);
-  return { stories, loading: fetching, onNextPage: onNextPageRef.current };
+  const handleReaction = useCallback(
+    async ({
+      targetId,
+      reactionName,
+      isLiked,
+    }: {
+      targetId: string;
+      reactionName: string;
+      isLiked: boolean;
+    }) => {
+      try {
+        if (isLiked) {
+          await ReactionRepository.removeReaction(
+            'story',
+            targetId,
+            reactionName
+          );
+        } else {
+          await ReactionRepository.addReaction('story', targetId, reactionName);
+        }
+      } catch (error) {
+        Alert.alert('Error on react');
```

Comment:
maybe better if we show alert with some useful infomation that throw from sdk if have any error

Reply:
Agree with @chaiwattsw

Reply:
Added sdk error in Alert krub